### ISSUE TITLE
compiling Xamarin.Forms Android App fails from network share

### ISSUE THREAD

**Issue by MagicAndre1981** [xamarin/xamarin-android#5218]
### Steps to Reproduce

1. [share your local drive](https://support.microsoft.com/en-us/help/4092694/windows-10-file-sharing-over-a-network)
2. open the .sln with VS2019 from network share and not from local drive
3. try to compile the android app

### Expected Behavior

app compiles fine

### Actual Behavior

app fails to compile:

```
System.IO.DirectoryNotFoundException: Could not find a part of the path 'D:\foo.app\obj\Debug\foo.App.Android\AnyCPU\lp\1\jl\__res_name_case_map.txt'
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalDelete(String path, Boolean checkHost)
   at System.IO.File.Delete(String path)
   at Xamarin.Android.Tools.Files.CopyIfStreamChanged(Stream stream, String destination)
   at Xamarin.Android.Tools.Files.ExtractAll(ZipArchive zip, String destination, Action`2 progressCallback, Func`2 modifyCallback, Func`2 deleteCallback, Func`2 skipCallback)
   at Xamarin.Android.Tasks.ResolveLibraryProjectImports.Extract(IDictionary`2 jars, ICollection`1 resolvedResourceDirectories, ICollection`1 resolvedAssetDirectories, ICollection`1 resolvedEnvironments)
   at Xamarin.Android.Tasks.ResolveLibraryProjectImports.RunTask()
   at Xamarin.Android.Tasks.AndroidTask.Execute()	foo.App.Android	C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Xamarin\Android\Xamarin.Android.Common.targets	1107	
```

this is a regression from VS2017, here the app compiles fine.

### Version Information

```
Microsoft Visual Studio Enterprise 2019
Version 16.7.6
VisualStudio.16.Release/16.7.6+30611.23
Microsoft .NET Framework
Version 4.8.03752

Xamarin   16.7.000.456 (d16-7-6@f8d55b6)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   16.7.0.495 (remotes/origin/d16-7@79c0c522c)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   16.7.85 (1bcbbdf)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   11.0.2.0 (d16-7/025fde9)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: 83105ba
    Java.Interop: xamarin/java.interop/d16-7@1f3388a
    ProGuard: Guardsquare/proguard/proguard6.2.2@ebe9000
    SQLite: xamarin/sqlite/3.32.1@1a3276b
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-7@017078f


Xamarin.iOS and Xamarin.Mac SDK   14.0.0.0 (7ec3751a1)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```

**Comment by jonathanpeppers** [comment]
I would not recommend building projects on a network share:

1. It is really slow.
2. Even after I fixed the above issue locally, [`aapt2.exe`](https://developer.android.com/studio/command-line/aapt2) hangs. We may or may not be able to fix this, as we have seen problems with aapt2 and network shares/external drives on Windows in the past.

I will still send a PR to fix the `DirectoryNotFoundException` above. It could help other situations I think.

**Comment by MagicAndre1981** [comment]
it works fine in VS2017 for years **without any issues**. The reason is because I use VMs to run Visual Studio and let the code on a host SSD to avoid code loss if VM gets corrupted

**Comment by jonathanpeppers** [comment]
>  I use VMs to run Visual Studio and let the code on a host SSD to avoid code loss if VM gets corrupted

I think source control like git is a better solution than this.

Your situation may continue to work using aapt (turning off aapt2). aapt is deprecated in the next release:

https://github.com/xamarin/xamarin-android/blob/master/Documentation/release-notes/aapt.md

Even so, I still wouldn't do this.

**Comment by MagicAndre1981** [comment]
I use Subversion, but if VM gets corrupted before committing the data I loose my work and have to do it again. 

I have to use aapt2 as aapt causes [errors with AndroidX](https://github.com/xamarin/AndroidX/issues/154)

**Comment by MagicAndre1981** [comment]
I've [tested the Xamarin.Android.Sdk-OSS-11.1.99.179_remotes-pull-5221-merge_949267d.vsix](https://github.com/xamarin/xamarin-android/pull/5221#issuecomment-712667648) and beside the initial missing `system.codedom.dll`, this version works fine for me.

Will it be part of 16.8 or 16.9?

**Comment by jonpryor** [comment]
> Will it be part of 16.8 or 16.9?

This fix should be included in 16.9 Preview 2.

**Comment by MagicAndre1981** [comment]
> This fix should be included in 16.9 Preview 2.

thanks

**Comment by MagicAndre1981** [comment]
Works fine in VS 16.9 release
